src/arkode/arkode_io.c

```diff
@@ -1620,11 +2345,32 @@ int arkGetCurrentState(void* arkode_mem, N_Vector* state)
 }
 
 /*---------------------------------------------------------------
-  arkGetCurrentTime:
+  ARKodeGetEstLocalErrors:
+
+  Returns an estimate of the local error
```

Comment:
No.  In the MRI adaptivity branch, I'm planning to support the use of H-M adaptivity (as Alex had introduced in his thesis) -- this requires that the inner stepper use fixed step sizes while also providing an estimate of the temporal error.  

It might be nice if this were added to the v-table, so that each stepper could either provide the routine or not, and could then internally check to see whether it had been configured to even compute that local error estimate.  Thoughts?

Reply:
I think adding this function to the vtable makes sense